### ISSUE TITLE
Using print screen key also sends alt key

### ISSUE THREAD

**Issue by the-wes** [symless/synergy-core#4288]
When using the PrintScreen key via Synergy to capture a screenshot on a Windows client, Synergy always transmits the key as Alt+PrintScreen, which tells Windows to capture only the active application window rather than the full screen.

See this example: http://gfycat.com/PlumpWindyBeaver

**Comment by virtualdxs** [comment]
Has this issue been fixed? I see it's still open and the PR hasn't been merged but this hasn't been touched since 12/23/**2014**.

**Comment by themarkwilliams** [comment]
Any updates on this. . .would love to see this fixed.

**Comment by nlyan** [comment]
Should be fixed in v1.8.8-rc1

Give it a go. http://symless.com/nightly?filter=4b913b5

**Comment by EvzenP** [comment]
Hmmm, I just tried v1.8.8-rc1 and I see no change - still only active window gets printscreened

**Comment by Banderi** [comment]
Can confirm, the bug is still present.

**Comment by EvzenP** [comment]
Okay, only now looking through the source code, even with my non-developer limited knowledge I see that the patched piece was definitely not the right place, i.e. this piece of code does not seem to 'convert' the PrintScreen to Alt-PrintScreen.
It's probably somwhere here:
https://github.com/symless/synergy/blob/master/src/lib/platform/MSWindowsKeyState.cpp#L1214
or here:
https://github.com/symless/synergy/blob/master/src/lib/platform/MSWindowsKeyState.cpp#L1250

**Comment by yupi2** [comment]
I decided to actually get a PR upstream for this here #5885 since I've just been sitting on it for 3 months. Did not intend to submit the PR right now but the Enter button was hit and it was posted... If anyone has a build environment for Windows and wants to test then check it out.

**Comment by dmittleider** [comment]
I just downloaded 1.9.1 and this is still an issue.  Is there a test build where this works at??

**Comment by dmittleider** [comment]
I downloaded the pre-compiled one from https://github.com/yupi2/synergy/releases/tag/4.2.0_master_0d1aad0dacc071f9981f778e2ee8a10f36b57ec5 but gives me a Qt error.  Are there any other pre-compiled ones for Windows x64?

**Comment by dmittleider** [comment]
OK! Perfect, after running it a couple times that went away and the Alt+PrntScrn is now just PrntScrn... I can not tell you how awesome this is.  I've used Synergy for years off and on (I come back to it and then get frustrated that doesn't work because my life is doing a ton of documentation and bringing screenshots in).  I tried it again here the other day and thought somebody has had to have solved this by now...  Thanks!!!

**Comment by Banderi** [comment]
Is there any way to switch to that version without losing server/client settings? Someone had the bright idea of dropping the config file, so I had no idea where to look for it and lost all of them.

**Comment by yupi2** [comment]
@Banderi I didn't like where Synergy stored stuff and wanted certs&configs to be stored in the same directory along with some stuff needing to be changed so it doesn't conflict with Synergy from Symless (registry keys & file paths). It's a pain but if you can find out where SymlessSynergy stores files then you can likely just drop them into one of the following folders (from the changes in the readme):

+ Moved configuration files location for no good reason.
  + Windows uses `%localappdata%\Synergy\`
  + macOS uses `~/Library/Preferences/Synergy/`
  + Linux/everything-else uses `~/.config/Synergy/`

The [Everything](https://www.voidtools.com/) program is a good tool for finding files. Searching for `Synergy` will probably show you all the folders & files related to everything.

My synergy folder looks like this
```
PS C:\Users\yupi2\AppData\Local\Synergy> tree /F
Folder PATH listing
Volume serial number is ASDF
C:.
│   gui.ini
│
└───SSL
    │   Synergy.pem
    │
    └───Fingerprints
            Local.txt
```

**Comment by Banderi** [comment]
I could swear I tried searching for the config files in AppData but I could find nothing. I very much agree the registry is an asinine idea and I totally endorse your decision of using same-folder config files.

**Comment by themarkwilliams** [comment]
What's the resolution of this issue? 

Is it fixed? If so, which version contains the fix?

**Comment by yupi2** [comment]
Works for me with my pull-request here https://github.com/symless/synergy-core/pull/5885
https://github.com/symless/synergy-core/pull/5885.patch
Someone at Symless would just need to review it and apply it.
@nbolton

**Comment by nbolton** [comment]
@themarkwilliams It was assigned to 1.8.8 but was reopened, so the fix in fact didn't work (and it should never have been assiged to 1.8.8 in the first place). The comments seem to suggest that it is actually not fixed, as commit 3d3b7ca8810f283d2134cdb3fc92b77ef0233c1b wasn't effective.

@yupi2 I'll make a build for your PR (#5885) and post it on this issue to see if it helps.

**Comment by hansdg1** [comment]
Any progress on this?

**Comment by nbolton** [comment]
Fixed by #6618
